From 88784e114e2b02552ef797d2f61eecaee31eda86 Mon Sep 17 00:00:00 2001
From: Nikola Mihaylov <nmihaylov@mm-sol.com>
Date: Wed, 14 Aug 2013 14:51:08 +0300
Subject: [PATCH 28/44] audio: codec: wcd9310: Added AMIC - LDOH connections.

	Added LDOH connection for every AMIC. This
	will ensure that path from inputs to SlimBus
	is correct and functional.

Change-Id: I0c65aaf424a2cee78c94fe9a5c5a58abde0eb92c
Signed-off-by: Nikola Mihaylov <nmihaylov@mm-sol.com>
---
 sound/soc/codecs/wcd9310.c |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/sound/soc/codecs/wcd9310.c b/sound/soc/codecs/wcd9310.c
index 5ddaf7b..328bcf5 100644
--- a/sound/soc/codecs/wcd9310.c
+++ b/sound/soc/codecs/wcd9310.c
@@ -3484,6 +3484,14 @@ static const struct snd_soc_dapm_route audio_map[] = {
 	{"ADC5", NULL, "AMIC5"},
 	{"ADC6", NULL, "AMIC6"},
 
+	/* AMIC - LDOH Connetcions */
+	{"AMIC1", NULL, "LDO_H"},
+	{"AMIC2", NULL, "LDO_H"},
+	{"AMIC3", NULL, "LDO_H"},
+	{"AMIC4", NULL, "LDO_H"},
+	{"AMIC5", NULL, "LDO_H"},
+	{"AMIC6", NULL, "LDO_H"},
+
 	/* AUX PGA Connections */
 	{"HPHL_PA_MIXER", "AUX_PGA_L Switch", "AUX_PGA_Left"},
 	{"HPHL_PA_MIXER", "AUX_PGA_R Switch", "AUX_PGA_Right"},
-- 
1.7.9.5

